setup.py

```diff
@@ -1,6 +1,8 @@
 # coding: utf-8
-
-from distutils.core import setup
+try:
```

Comment:
I think nowadays it's safe to assume that everybody will have setuptools installed. I would prefer removing this conditional.